src/main/java/com/kalessil/phpStorm/phpInspectionsEA/inspectors/apiUsage/arrays/LowPerformanceArrayUniqueUsageInspector.java

```diff
@@ -12,8 +12,8 @@
 import org.jetbrains.annotations.NotNull;
 
 public class LowPerformanceArrayUniqueUsageInspector extends BasePhpInspection {
-    private static final String strProblemUseArrayKeysWithCountValues = "array_keys(array_count_values(<expression>)) will be more performing (but provide comments)";
-    private static final String strProblemUseCountWithCountValues     = "count(array_count_values(<expression>)) will be more performing (but provide comments)";
+    private static final String strProblemUseArrayKeysWithCountValues = "array_keys(array_count_values(...)) would be more efficient.";
```

Comment:
The point is that we're suggesting to make code less readable here, so fair would be to put some obligations to the reader. Then, the reader will think a little bit if it's really worth it.